tests/test_integration.py

```diff
@@ -1,6 +1,14 @@
+import sys
+
+import pytest
+
 from twine import cli
 
 
+@pytest.mark.xfail(
```

Comment:
If we know they're going to fail, could we [skip](https://docs.pytest.org/en/stable/skipping.html#id1) them?

```suggestion
@pytest.mark.skipif(
```

Reply:
To clarify: [xfail](https://docs.pytest.org/en/stable/skipping.html#xfail-mark-test-functions-as-expected-to-fail) still run the tests.

Reply:
If we skip them, we'll never know if they start passing. xfail will report if they start passing